pkgs/build-support/fetchurl/default.nix

```diff
@@ -124,5 +134,12 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
   # traffic, so don't do that.
   preferLocalBuild = true;
 
+  inherit fetchType;
+  fetchArguments = {
+     url = head urls;
+     hash = outputHash;
+     type = outputHashAlgo;
+  };
```

Comment:
I just wanted to collect all data that find-tarballs.nix should pass on to copy-tarballs.pl. I haven't checked whether `derivation` fails, thanks. Maybe I should just generate a list of attribute names?